dockerfiles/reverse-proxy/README.md

```diff
@@ -0,0 +1,13 @@
+# reverse-proxy
+
+The reverse-proxy Dockerfile is the Barcelona supported reverse proxy (based on nginx) that is configured to cleanly integrate with the other Barcelona images.
```

Comment:
Apologies if this is incorrect, I guessed a little 😄 . If there's a better description for the image let me know and I'll update the PR 😄

Reply:
I don't know the original intent of the reverse proxy but I will assume its the standard fare of, we want to have our own nginx.conf to modify so we can do things the ELB does not let us do with traffic before they flow to our server.